tests/test_flow/test_outputs.py

```diff
@@ -123,10 +123,21 @@ def three_outputs():
         flow.get("a")
 
 
+def test_non_sequence_output(builder):
```

Comment:
nit: I feel like `test_non_sequence_output` suggests that we're testing `@output` rather than `@outputs`.

Reply:
Addressed your comment about checking the exception message and changed the name to `test_non_sequence_outputs_message` since there's already a `test_non_sequence_outputs`.

Reply:
Changed this to `test_non_sequence_outputs_message` since there's already a `test_non_sequence_outputs`, also addressing the comment above.